app/controllers/v0/mvi_users_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module V0
+  class MviUsersController < ApplicationController
+    before_action { authorize :mvi, :access_add_person? }
+
+    def submit
+      # Caller must be using proxy add in order to complete Intent to File or Disability Compensation forms
+      form_id = params[:id]
+      if [FormProfiles::VA0996::FORM_ID, VA526ez::FORM_ID].exclude?(form_id)
+        raise Common::Exceptions::Forbidden.new(
+          detail: "Action is prohibited with id parameter #{form_id}",
+          source: 'MviUsersController'
+        )
+      end
+
+      # Scenario indicates serious problems with user data
+      if @current_user.birls_id.nil? && @current_user.participant_id.present?
+        raise Common::Exceptions::UnprocessableEntity.new(
+          detail: 'No birls_id while participant_id present',
+          source: 'MviUsersController'
+        )
+      end
+
+      # Add user to MVI
```

Comment:
So, in this scenario, the person may already be in MVI, but just have id's for other sytsems I think it'd be nice to make that clear in this comment.

Reply:
@annaswims Are you just asking me to be more specific? E.g. write a one shorter version of the swagger description >> https://github.com/department-of-veterans-affairs/vets-api/pull/4384/files#diff-f2e2eff8088c7aa0ac8d81ead95df70aR16-R21